Yoast/wordpress-seo
# Adds the possibility to reindex all indexables.

## Context


* We want to provide the possibility to reindex all the indexables by using WP CLI

## Summary


This PR can be summarized in the following changelog entry:

* Adds a flag (`reindex`) to reindex all the indexables.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Perform this command `wp yoast index --reindex` in the CLI.
    * On Docker installations, you should run `./wp.sh yoast index --reindex` in the `plugin-development-docker` directory. 
* Everything should be reindexed. You can edit values in the indexables database table to see it has been changed after running to command.

## UI changes

## Documentation

## Quality assurance


Fixes #14908
